server/src/main/scala/io/delta/kernelsharedtable/DeltaSharedTableKernel.scala

```diff
@@ -114,6 +119,89 @@ class DeltaSharedTableKernel(
     (table, engine)
   }
 
+  // Get the table snapshot of the specified version or timestamp
+  // TODO: Validate the protocol and metadata
+  private def getSharedTableSnapshot(
+      table: Table,
+      engine: Engine,
+      version: Option[Long],
+      timestamp: Option[String],
+      validStartVersion: Long,
+      clientReaderFeatures: Set[String],
```

Comment:
I would rather keep the stub for this here right now since I made the structural change to DeltaSharingService, and this will indicate where all I need to consider the clientReaderFeatures for future work. Since it is not being used, it will not lead to any code path not being tested

Reply:
sg